### ISSUE TITLE
[Feature Request] Visualizing the evaluations should look different from the promptflow traces, should provide some kind of data visualization

### ISSUE THREAD

**Issue by tyler-suard-parker** [microsoft/promptflow#3492]
**Is your feature request related to a problem? Please describe.**
Right now, when we visualize the evaluations, it is not easy to understand the results.  For example the result of visualizing in this notebook promptflow\examples\flex-flows\chat-async-stream\chat-stream-with-async-flex-flow.ipynb looks like this:

<img width="930" alt="image" src="https://github.com/microsoft/promptflow/assets/147888261/2efee013-866c-4450-820e-5705553a4a7c">

It is not easy to see which evals failed and which succeeded, or a proportion of successes vs failures.

**Describe the solution you'd like**
It would be nice to have a clearer visualization for the evaluations, because their purpose is different from the traces.  For an evaluation we usually just want a simple pass/fail, whereas with a trace we want the full details.  Here is an example:

[eval report.zip](https://github.com/user-attachments/files/16073061/eval.report.zip)

**Comment by zhengfeiwang** [comment]
Thank you for your suggestion! Add screenshot of your example below:

![image](https://github.com/microsoft/promptflow/assets/38847871/c452f932-a5c3-44ab-a441-33b6a0d36418)

@tyler-suard-parker one thing I'd like to confirm: in which step you get the above trace UI page? I see there are two runs in the url, so I guess you are getting this from the line `pf.visualize([base_run, eval_run])`?

If so, how about changing it to `pf.visualize(base_run)` and see if it looks better? The evaluation run's result will be appended to the corresponding line - maybe we should update our notebook there, `pf.visualize` is something different before and switch to leverage trace UI recently

**Comment by tyler-suard-parker** [comment]
I tried running pf.visualize(base_run) and I got this.  When I enabled the metrics column it looks a little better, but there is still a lot of information I don't need if I'm doing an evaluation:
<img width="941" alt="image" src="https://github.com/microsoft/promptflow/assets/147888261/b69dfdb0-1f45-4b41-8733-0cf5bbc8e2fa">

I use evaluations as unit tests for my prompt engineering.  I have 10 standard questions I ask.  Every time I change one of my agent prompts, I run those 10 standard questions again as part of my CI/CD tests and I look at the test report, just to make sure none of my changes caused any incorrect answers.  When I'm doing this with every commit, I don't have time to read through all the traces.  It would be nice to just have a single diagram that shows me how the evaluation batch went.
